package.json

```diff
@@ -9,16 +9,18 @@
     "middleware",
     "spring"
   ],
+  "types": "./types",
```

Comment:
@sioked I think you need to actually specify the definition file, so `./types/index.d.ts`

Reply:
I followed the instructions here: https://github.com/Microsoft/dtslint, I think maybe etiher would work since the file is index.d.ts? Maybe we test once its published....